test/helper-async.js

```diff
@@ -82,7 +82,7 @@ function pm(args, env, callback) {
     console.log('pmcli:', pmcli, args);
     cleanup = once(cleanup);
 
-    pm = cp.spawn(pmcli, args, {
+    pm = cp.fork(pmcli, args, {
```

Comment:
I recall you were suggesting using exec instead of spawn followed by a slurp of the stdout/err, but spawn allowed an ipc channel to be attached, so children would cleanup.

The `stdio` option below should be deleted, I know node ignores it, but its confusing to humans.